src/js/flags/getFlagForName.ts

```diff
@@ -54,3 +54,5 @@ const getFlagFromCollection =
     (flags: Flag[], searchFlag: string): Flag|undefined => {
       return flags.find(flag => flag.flag === searchFlag);
     }
+
+    
```

Comment:
Okay this one is really weird...

Reply:
I added some extra spaces before I realized I was ignoring the white space during compares. That's intentional.